resources/.eslintrc-react.js

```diff
@@ -52,6 +72,8 @@ module.exports = {
          */
         "react/jsx-boolean-value": ["error", "always"],                               // Enforce boolean attributes notation in JSX (fixable)
         "react/jsx-closing-bracket-location": ["error", {"location": "after-props"}], // Validate closing bracket location in JSX (fixable)
```

Comment:
I wouldn't mind seeing this one use the default locations:

```js
"react/jsx-closing-bracket-location": ["error"]
```

It's the prevailing convention and avoids creating blank lines for the purpose of making nesting clearer.

Currently we do this:

```jsx
<Parent
    foo={1}
    bar="xyz">
    <Child />
</Parent>
```

which unfortunately aligns the parent's props with the child, so we often end up doing this:

```jsx
<Parent
    foo={1}
    bar="xyz">

    <Child />

</Parent>
```

With the default settings, we can do this instead:

```jsx
<Parent
    foo={1}
    bar="xyz"
>
    <Child />
</Parent>
```

Also, we do this:

```jsx
<SelfClosing
    a={z}
    b="abc" />
```

but most people do this:

```jsx
<SelfClosing
    a={z}
    b="abc"
/>
```

and I think it looks nicer.

Reply:
Agreed, AirBnB’s linting rules has this, and I liked its readability. It’s also a fixable linting error, so easy to update a repo with this new rule style. Anyone else want to weigh in?

Reply:
I don't mind the `>` position on the opening tag. Not sure why we landed on that to begin with. 

The line breaking the self closing tag looks a bit weird to me though. 

```
<Parent
    foo={1}
    bar="xyz"
>
    <Child 
        foo={1}
        bar="xyz" /> // "inline" seems a bit more intuitive for me here
</Parent>
```

Reply:
Is a bit weird, yes, but I think when you’re typing it over and over you get used to hitting enter after your last attribute & value, so having one habit of hitting enter to close the tag and one habit of not hitting enter to close the tag seems like more cognitive load :)

Reply:
It's pretty much JSX convention. I've yet to come across code examples that do it our way.

e.g. https://reactjs.org/#todoExample

Reply:
JSX is one thing but we need to ensure that our HTML/XML follows the same standard too.

Anyway I really don't mind, so if it makes the team happy then I'm happy. Put it in the PR and get enough 👍  and lets move on